### PR TITLE
Fixes for 1.4.0

### LINKED ISSUE #177: Don't reexport 'mask_'
This function is too low-level.

### LINKED ISSUE #178: Remove 'getArgs' from 'Lifted.Env'
Probably there's no much sense in having lifted `getArgs`. 

Also, let's think about `exit*` functions. Do we need them all? At least it's strange to export function with type:

```haskell
exitWith :: MonadIO m => ExitCode -> m a
```

but not export `ExitCode` type.

### LINKED ISSUE #179: Remove 'getContents' and 'interact' functions
They are not used. And probably it's not a good idea to use them.

### LINKED ISSUE #180: Remove 'Lifted.ST' module
Nobody uses it.

### LINKED ISSUE #181: Remove 'list' function
It's weird. And it stops from having `list` variable name 😞

### LINKED ISSUE #192: Delete `identity`
It was deprecated in 1.3.0, let's delete it in the next breaking release.

### PR DESCRIPTION
**Pull request by rockbmb**
Closes #177, closes #178, closes #179, closes #180, closes #181, and closes #192.

### CODE REVIEW

**Comment by rockbmb** [comment]
To be reviewed in September 2018.

**Comment by rockbmb** [comment]
@gromakovsky September has come 😄

File: CHANGES.md [hunk 1]
```diff
@@ -4,6 +4,16 @@
   `identity` has been removed.
 
   _Migration guide:_ use `Universum.id` instead.
+* [#178](https://github.com/serokell/universum/issues/178):
+  `getArgs` has been removed.
+
+  _Migration guide:_ use `liftIO` directly with `System.Environment.getArgs`
+  from base.
+* [#179](https://github.com/serokell/universum/issues/179):
```
Comment (gromakovsky) [inline CHANGES.md]: Nitpick: it's in the commit which removes `getArgs`, can you move it to the commit where this change (removal of `getContents` and `interact`) is actually done?

**Comment by rockbmb** [comment]
@gromakovsky fixed 👍. As for reviews, @volhovm взгляни, пожалуйста.

**Comment by rockbmb** [comment]
@Pastafarianist Mind taking a look as well? Won't take too long, and it'll help move `universum` forward.

**Comment by gromakovsky** [comment]
AFAIK, @volhovm is moving to Paris, that's why he's inactive these days. He was going to review current PRs this week though.

**Comment by rockbmb** [comment]
@volhovm The only occurences of the relevant functions occurs in `CHANGES.md`, either to inform they were added in a previous version, or to inform they are going to be removed in 1.4.0.

**Comment by gromakovsky** [comment]
@rockbmb please rebase on latest `master` and let's merge this PR.

**Comment by rockbmb** [comment]
@gromakovsky done.

**Pull request merged**